script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector(".todo-form");
+const toDoInput = toDoForm.querySelector("input");
+const toDoInputBtn = document.querySelector(".todo-input").querySelector("div");
+const toDoListPre = document.querySelector(".todo-list-pre");
+const toDoListDone = document.querySelector(".todo-list-done");
+
+let toDoList = [];
+let indPre = 0;
+let indDone = 0;
```

Comment:
```suggestion
let idxPre = 0;
let idxDone = 0;
```
변수명은 정확히 사용하거나, 통용되는 줄임말을 사용하는 것이 좋겠죠.

Reply:
넵!! 앞으로의 코드들에도 idx를 사용하도록 하겠습니다